packages/react-core/src/core/utils/updateChainRpcs.ts

```diff
@@ -5,8 +5,12 @@ import { getChainRPCs, ChainRPCOptions, Chain } from "@thirdweb-dev/chains";
  * use the keys and return a new chain object with updated RPCs
  */
 export const updateChainRPCs = (chain: Chain, options: ChainRPCOptions) => {
-  return {
-    ...chain,
-    rpc: getChainRPCs(chain, options),
-  };
+  try {
+    return {
+      ...chain,
+      rpc: getChainRPCs(chain, options),
```

Comment:
Do you know why is this throwing?

Reply:
@iketw throws when passing a `Chain` with empty `rpc` array

Reply:
In dashboard, we are gonna pass all 700+ chains as supportedChains and some of them have empty rpc array